lumicks/pylake/correlated_stack.py

```diff
@@ -49,6 +49,7 @@ def __init__(self, *image_names, align=True):
         )
         self.start_idx = 0
         self.stop_idx = self.src.num_frames
+        self.step = 1
```

Comment:
Please don't forget to make it private as well (preface with underscore) :)

Reply:
Done. Shouldn't we also make start_idx and stop_idx private, then?

Reply:
Probably yes. Should probably make it private and add an entry in the changelog about it (since it is technically a breaking change).

Reply:
Done